server/php-worker.cpp

```diff
@@ -57,6 +56,11 @@ double PhpWorker::enter_lifecycle() noexcept {
     get_utime_monotonic();
   } while (!paused);
 
+  if (conn->status != conn_wait_net) {
+    vkprintf(0, "Connection suspended without waiting for net. PhpWorker state %d, connection status %d, "
+                "PhpScript state %d\n", state, conn->status, php_script != nullptr ? static_cast<int>(php_script->state) : -1);
+
+  }
   assert(conn->status == conn_wait_net);
```

Comment:
I think it worths merging these errors into one in sentry, with all necessary info